File: Models/Entities/Burger.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace BrasilBurger.Web.Models.Entities
{
    [Table("burgers")]
    public class Burger
    {
        [Key]
        [Column("id")]
        public int Id { get; set; }

        [Required]
        [Column("nom")]
        [StringLength(100)]
        public string Nom { get; set; }

        [Column("description")]
        public string? Description { get; set; }

        [Column("prix")]
        public decimal Prix { get; set; }

        [Column("image")]
        public string? Image { get; set; }

        // Add other properties as needed
    }
}

File: BrasilBurger_Java/src/main/java/com/brasilburger/model/Burger.java
package com.brasilburger.model;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.math.BigDecimal;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class Burger {
    private int id;
    private String nom;
    private BigDecimal prix;
    private String image;
    private String description;
    private boolean isArchived;
}
